pkg/graph/schema.graphql

```diff
@@ -1226,6 +1230,39 @@ enum SystemIntakeGRBReviewerVotingRole {
   NON_VOTING
 }
 
+type SystemIntakeGRBReviewDiscussionPost {
+  id: UUID!
+  content: HTML!
+  votingRole: SystemIntakeGRBReviewerVotingRole
+  grbRole: SystemIntakeGRBReviewerRole
+  systemIntakeID: UUID!
+  createdByUserAccount: UserAccount!
+  createdAt: Time!
+  modifiedByUserAccount: UserAccount
+  modifiedAt: Time
+}
+
+type SystemIntakeGRBReviewDiscussion {
+  initialPost: SystemIntakeGRBReviewDiscussionPost!
+  replies: [SystemIntakeGRBReviewDiscussionPost!]!
+}
+
+
+input createSystemIntakeGRBDiscussionPostInput {
+  systemIntakeID: UUID!
+  content: HTML!
+}
+
+input createSystemIntakeGRBDiscussionReplyInput {
+  initialPostID: UUID!
+  content: HTML!
+}
+
+type CreateSystemIntakeGRBDiscussionPostPayload {
+  post: SystemIntakeGRBReviewDiscussionPost
+  userErrors: [UserError!]
+}
```

Comment:
I don't think we ever _really_ use this `Payload`/`UserError` type in most cases. I'd be happy to just simplify the response wherever this is used to instead just be a `SystemIntakeGRBReviewDiscussionPost`

Also fine to keep it as-is, but I don't feel any pressure to use this structure IMO!

Reply:
Changed in [2e29e38](https://github.com/CMS-Enterprise/easi-app/pull/2887/commits/2e29e38318ae8e416e7e725efd5eb01130c8cb31)